opensrp-anc/src/main/res/values-pt_BR/strings.xml

```diff
@@ -1,420 +0,0 @@
-<resources>
```

Comment:
Why have we deleted this file?

Reply:
this file was causing build errors as there were two pt-BR string files system does not recognize pt_BR string it recognizes pt-rbr instead